src/App.jsx

```diff
@@ -1,60 +1,293 @@
-import "./App.css"
+import "./App.css";
+import { useState } from "react";
 
 function App() {
+  const [leftCalcValue, newLeftCalcValue] = useState("0");
+  const [rightCalcValue, newRightCalcValue] = useState("0");
+  const [operator, newOperator] = useState("+");
+  const [result, newResult] = useState(0);
 
   return (
     <div className="calculator">
       <div className="panel">
-        <p>0</p>
+        <p>{leftCalcValue}</p>
         <div className="numbers">
-          <button>1</button>
-          <button>2</button>
-          <button>3</button>
-          <button>4</button>
-          <button>5</button>
-          <button>6</button>
-          <button>7</button>
-          <button>8</button>
-          <button>9</button>
-          <button>0</button>
-          <button>Clear</button>
+          <button
+            onClick={() => {
+              if (leftCalcValue === "0") {
+                newLeftCalcValue("1");
+              } else {
+                newLeftCalcValue(leftCalcValue + "1");
+              }
+            }}
+          >
+            1
+          </button>
+          <button
+            onClick={() => {
+              if (leftCalcValue === "0") {
```

Comment:
you must organise all of these as functions that get called!!

eg.
```js
const addLeftNum = (numToAdd) => { ... }

// in JSX:
onClick={() => addLeftNum("5")}
```

You are repeating the code too many times... Functions help you achieve code re-use.